src/index.ts

```diff
@@ -16,6 +18,12 @@ const run = async () => {
 
   const database = mongoClient.db();
 
+  const authController = new AuthController(database.collection('users'));
```

Comment:
stworz usersCollection jako zmienna i dopiero przekaz, bedzie to bardziej uporzadkowane.